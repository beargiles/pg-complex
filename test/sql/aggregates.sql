\set ECHO None
BEGIN;
\i sql/complex.sql
\set ECHO all
   
--
-- check aggregates
--
-- FIXME
--

ROLLBACK;
