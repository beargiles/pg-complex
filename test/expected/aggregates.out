\set ECHO None
   
--
-- check aggregates
--
-- FIXME
--
ROLLBACK;
